histomicstk/__init__.py

```diff
@@ -19,4 +21,5 @@
     'preprocessing',
     'segmentation',
     'utils',
+    'cli',
```

Comment:
Will this make the installation of ctk_cli a requirement? Could we not expose it until we figure out how to put ctk_cli on pypi or build wheels for it as part of our pypi deployment.

Reply:
No.  ctk_cli isn't a requirement unless you import one of the cli main modules (e.g., `from histomicstk.cli.NucleiDetection import NucleiDetection`).  If you do that, and have not installed ctk_cli, you'll get an import error.

Reply:
To be clear, you can even do `import histomicstk.cli.NucleiDetection` and you won't trigger the import error.